### ISSUE TITLE
Add Frame Stepper ( Advanced Frame ) ...

### ISSUE THREAD

**Issue by ThienGMD** [FireMario211/Prism-Menu#138]
-Add Frame Stepper, Auto Click P1 P2 P1&P2 to Quarzt Bot. Made accuracy pls
-Add Text Length Bypass

**Comment by FireMario211** [comment]
1. Frame Stepper already exists, its called Macro Editor
2. I know I haven't made commits yet but auto clicker will be in the next update (video below)
https://github.com/user-attachments/assets/2b387e4a-1c7b-4b6a-ad05-a5efcfeed5e4
3. Text Length Bypass was in before but was removed because it crashed Android users

**Comment by HiimAndrew123** [comment]
"Frame Stepper already exists, its in the Macro Editor"
hes talking about frame stepper while botting, not in the macro editor
asked for the same thing in the prism menu server and firee told me the same thing
bruh

**Comment by FireMario211** [comment]
> "Frame Stepper already exists, its in the Macro Editor" hes talking about frame stepper while botting, not in the macro editor asked for the same thing in the prism menu server and firee told me the same thing bruh

okay fine, ill add frame stepper even though it already technically exists

**Comment by HiimAndrew123** [comment]
> > "Frame Stepper already exists, its in the Macro Editor" hes talking about frame stepper while botting, not in the macro editor asked for the same thing in the prism menu server and firee told me the same thing bruh
> 
> okay fine, ill add frame stepper even though it already technically exists

not really but cool

**Comment by ThienGMD** [comment]
> 1. Frame Stepper already exists, its in the Macro Editor
> 2. I know I haven't made commits yet but auto clicker will be in the next update (video below)
>    https://github.com/user-attachments/assets/2b387e4a-1c7b-4b6a-ad05-a5efcfeed5e4
> 3. Text Length Bypass was in before but was removed because it crashed Android users

Its Complicated

**Comment by FireMario211** [comment]
not quite, only auto clicker was added, i still need to add frame stepper
